NEWS

```diff
@@ -3,7 +3,65 @@
  *
  * See file LICENSE for terms.
  */
-## 1.0.0 (TBD)
+
+## Current
+
+## 1.1.0 (TBD)
+
+## Features
+
+## API
+- Added float 128 and float 32, 64, 128 (complex) data types
+- Added Active Sets based collectives to support dynamic groups as well as
+  point-to-point messaging
+
+## Core
+- Config file support
+- Fixed component search
+
+## CL
+
+- Added split rail all reduce collective implementation
+- Enable hierarchical alltoallv
+- Fixed cleanup bugs
+
+
+## TL
+- Added SELF TL supporting team size one
+
+### UCP
+
+- Added service broadcast
+- Added reduce_scatterv ring algorithm
+- Added k-nomial based gather collective implementation
+- Added one-sided get based algorithms
+
+### SHARP
+- Fixed SHARP OOB
+- Added SHARP broadcast
+
+
+
+### GPU Collectives (CUDA, NCCL TL and RCCL TL)
+- Added support for CUDA TL (intranode collectives for NVIDIA GPUs)
+- Added multiring allgatherv, alltoall in CUDA TL
```

Comment:
also reduce-scatter and reduce-scatterv multiring in TL/CUDA
+ topo based ring construction in TL/CUDA that maximizes BW